cmd/cdi-cloner/cloner_startup.sh

```diff
@@ -33,15 +40,24 @@ if [ "$obj" == "source" ]; then
     echo "cloner: Starting clone source"
     echo "cloner: creating fifo pipe"
     mkfifo $pipe_dir
-    echo "cloner: creating tarball of the image and redirecting it to $pipe_dir"
-    pushd $image_dir
-	#figure out the size of content in the directory
-    size=$(du -sb . | cut -f1)
-    echo $size
-    #Write the size to the pipe so the other end can read it.
-    echo "$size" >$pipe_dir
-    tar cv --sparse . >$pipe_dir
-    popd
+    if [ "$volumeMode" != "block" ]; then
+        echo "cloner: creating tarball of the image and redirecting it to $pipe_dir"
+        pushd $image_dir
+	    #figure out the size of content in the directory
+        size=$(du -sb . | cut -f1)
```

Comment:
@awles Can you please point me where we use the size we echo into the pipe, in the target pod? I guess it's somewhere in target_cloner.go but I can't find it. Also I  assume we cannot use this approach with block device. Can we? Do you have an idea what can be done instead?

Reply:
We use the size to determine the total to calculate the progress of the clone. That is the only reason. It is used in the CountingReader its transparent to the writer of the file.

Reply:
but size=$(du -sb . | cut -f1) won't work on /data/clone which is block device. right?

Reply:
no but you know how to get the size of the block device with lsblk, we do it in the importer.